app/models/import.rb

```diff
@@ -7,6 +7,8 @@ class Import < ApplicationRecord
   # of Administrate.
   has_many :data_imports, -> { none }, inverse_of: "import"
 
+  has_one_attached :redacted_pdf
```

Comment:
Is this on the `Import` instead of `Imports::Pdf` because of Administrate?

Reply:
Oops, I thought I put that on `Imports::Pdf`. Thanks for catching. I haven't fit this into Administrate yet, so I might have to move it back, but I would prefer it just in the Pdf model for now.

Reply:
fixed in 1543809da677e571b89847007074233a296b824e